deploy/crds/core_v1_storagecluster_crd.yaml

```diff
@@ -1442,4 +1445,4 @@ spec:
     schema:
       openAPIV3Schema:
         type: object
-        preserveUnknownFields: false
+        x-kubernetes-preserve-unknown-fields: true
```

Comment:
This is unrelated, but k8s no longer recognizes the preserveUnknown field and expects us to use x-kubernetes-.. field.